hr_recruitment_security/tests/test_access_rights.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import logging
+from odoo.exceptions import AccessError
+from odoo.tests.common import SavepointCase
+
+_logger = logging.getLogger(__name__)
+
+
+class TestHrRecruitmentSecurity(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestHrRecruitmentSecurity, cls).setUpClass()
+        # Users
+        users = cls.env["res.users"].with_context(no_reset_password=True)
+        # hr_officer_own_documents
+        group_name = "group_hr_recruitment_officer_own_documents"
+        cls.user_hr_officer_own_documents = users.create(
+            {
+                "name": "hr_officer_own_documents",
+                "login": "hr_officer_own_documents",
+                "email": "hr_officer_own_documents@example.com",
+                "groups_id": [
+                    (6, 0, [cls.env.ref(
+                        "hr_recruitment_security.%s" % group_name
+                        ).id])
+                ],
+            }
+        )
+        # hr_user
+        cls.user_hr_ruser = users.create(
+            {
+                "name": "hr_user",
+                "login": "hr_user",
+                "email": "hr_user@example.com",
+                "groups_id": [
+                    (6, 0, [cls.env.ref("hr_recruitment.group_hr_recruitment_user").id])
+                ],
+            }
+        )
+        # without_groups
+        cls.user_without_groups = users.create(
+            {
+                "name": "without_groups",
+                "login": "without_groups",
+                "email": "without_groups@example.com",
+                "groups_id": False,
+            }
+        )
+        # hr_job
+        cls.hr_job = cls.env["hr.job"].create(
+            {
+                "name": "hr_recruitment_security"
+            }
+        )
+        cls.env["hr.job"].create((
+            {
+                "name": "hr_recruitment_security",
+                "hr_responsible_id": cls.user_hr_ruser.id,
+            },
+            {
+                "name": "hr_recruitment_security",
+                "hr_responsible_id": cls.user_hr_officer_own_documents.id
+            }
+        ))
+        # hr_applicant
+        cls.hr_applicant = cls.env["hr.applicant"].create(
```

Comment:
Why splitting this one?

Reply:
I split it because later, check related to this record if is possible read. Now it's remove